tests/test.py

```diff
@@ -1701,19 +1607,19 @@ def checkSkip(self):
     def setUp(self):
         super().setUp()
 
-        def executor_with_port(port):
+        def executor_with_port(ssl_key, ssl_cert):
```

Comment:
if we changed signature and function meaning then we should rename the function to not confuse other readers.

Reply:
You are right!